langwatch/src/components/PeriodSelector.tsx

```diff
@@ -161,26 +159,18 @@ export function PeriodSelector({
               <Field.Root>
                 <Field.Label>Start Date</Field.Label>
                 <Input
-                  type="date"
-                  value={format(startDate, "yyyy-MM-dd")}
-                  onChange={(e) =>
-                    setPeriod(
-                      startOfDay(new Date(e.target.value + "T00:00:00")),
-                      endDate
-                    )
-                  }
+                  type="datetime-local"
+                  value={format(startDate, "yyyy-MM-dd'T'HH:mm")}
+                  onChange={(e) => setPeriod(new Date(e.target.value), endDate)}
```

Comment:
adding midnight was good to match user expectation that when they select e.g. 15/jan (a past date), it includes 15/jan from midnight, not from their local current time

is this still valid now? If I select a past date does it uses my current time (eg. 9am) as the initial time, or midnight by default?

Reply:
nvm I noticed on screenshot it's midnight